test/functional/feature_fee_estimation.py

```diff
@@ -129,6 +129,15 @@ def make_tx(wallet, utxo, feerate):
     )
 
 
+def send_tx(wallet, node, utxo, feerate):
```

Comment:
Nit: The meaning would be more explicit if we called the last argument as `feerate_satvb`.

Reply:
Their is a description below that states that the feerate is in sat/vb so I will leave this as is.